src/SFA.DAS.EmployerIncentives.Api/Authorization/NoneRequirement.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Microsoft.AspNetCore.Authorization;
+
+namespace SFA.DAS.EmployerIncentives.Api.Authorization
+{
+    public class NoneRequirement : IAuthorizationRequirement
```

Comment:
As per previous, looks like this could be shared?

Reply:
yes